### ISSUE TITLE
Custom headers render HTML as text regression

### ISSUE THREAD

**Issue by anssiko** [tabatkins/bikeshed#1097]
Custom headers render embedded HTML as text starting with 7662ed324a33038d0ccf9ddb6e473f076a7a5513

Works as expected in 6e949445d0006e32e94031287e7088aa812092a0 and earlier.

Given the following Bikeshed source:
```
!Feedback: <a href="https://github.com/w3c/sensors">GitHub</a> (<a href="https://github.com/w3c/sensors/issues/new">new issue</a>, <a href="https://github.com/w3c/sensors/milestone/2">level 1 issues</a>, <a href="https://github.com/w3c/sensors/issues">all issues</a>)
```
The rendered output is:
```
<a href="https://github.com/w3c/sensors">GitHub</a> (<a href="https://github.com/w3c/sensors/issues/new">new issue</a>, <a href="https://github.com/w3c/sensors/milestone/2">level 1 issues</a>, <a href="https://github.com/w3c/sensors/issues">all issues</a>)
```
Instead of expected:

<a href="https://github.com/w3c/sensors">GitHub</a> (<a href="https://github.com/w3c/sensors/issues/new">new issue</a>, <a href="https://github.com/w3c/sensors/milestone/2">level 1 issues</a>, <a href="https://github.com/w3c/sensors/issues">all issues</a>)

**Comment by tabatkins** [comment]
It doesn't for me? Neither with the current head, nor with 7662ed3 specifically. Both of them render the HTML in the expected way - as HTML.

(It would have been really hard for me to break this, given that I was *explicitly* testing with HTML in the custom metadata when making these edits, both for this reason and to make sure it didn't just spit out the first HTML element only.)

**Comment by tabatkins** [comment]
...huh, and yet, the Sensors spec definitely breaks. Let me see wtf is going on.

**Comment by tabatkins** [comment]
Oh gosh, it's because this was a custom metadata that appended onto a built-in one. Fixing now, and adding to tests.

**Comment by anssiko** [comment]
Much thanks for an amazingly fast response!

**Comment by tabatkins** [comment]
Fixed in 7ac83000 (or rather the previous commit, but I introduced another minor regression in that one, so 7ac83000 actually makes all tests pass ^_^).
